db/migrate/20231130105924_create_comments.rb

```diff
@@ -0,0 +1,15 @@
+class CreateComments < ActiveRecord::Migration[7.1]
+  def change
+    create_table :comments do |t|
+      t.bigint :author_id
```

Comment:
- [ ] According to the [ERD](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png), please keep in mind that the author_id column is only used with the posts table, and not in the comments and likes tables, please stick to the given ERD for better a development experience. 🙏